docs/source/operator_reference_load_apis.rst

```diff
@@ -0,0 +1,26 @@
+:orphan:
+
+=========
+Operators
+=========
+
+Loading operators.
+
+.. raw:: html
+
+    <iframe
+      src="_static/dpf_operators.html"
```

Comment:
@anslpa How are we sure of the name it will have at the end? I mean, that the name will be different to the one that is already there (under `operator_reference_load.rst`).

Reply:
@rafacanton that's the name of the RST file which is used